src/Integration.Vsix/SonarLintTagger/IssueMarker.cs

```diff
@@ -33,9 +33,9 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     internal class IssueMarker
     {
         public IAnalysisIssue Issue { get; }
-        public SnapshotSpan Span { get; }
+        public SnapshotSpan? Span { get; }
```

Comment:
I made the span nullable. I suspect the better alternative would be to create IIssueMarker with just IAnalysisIssue, and ITextBufferIssueMarker  that also has a SnapshotSpan.

Reply:
I think making the span nullable is ok; I don't think switching on type in multiple places is any better.

Adding an extension method that filters a list of issue markers that are "taggable" would be effectively the same as doing the `OfType` check.